### PR TITLE
Cloud Service - Expanding notifications

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Addresses [DOCS-740](https://enterprisedb.atlassian.net/browse/DOCS-740)

[DOCS-740]: https://enterprisedb.atlassian.net/browse/DOCS-740?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nidhibhammar** [comment]
Note to self - I just added rough content as the first initial draft.

File: advocacy_docs/edb-postgres-ai/console/using/notifications.mdx [hunk 1]
```diff
@@ -42,12 +47,77 @@ The notifications under the **Projects** preference section are:
...
+   1. Slack channel on which notification are to be sent. For example: #general, #alerts, #notifications, etc.
+   1. Select **Save** button
+
+!!!note
+Before configuring slack notification channel, you need to create the slack app and grant the following permissions:
+- `chat:write`
+- `im:write`
+- `users:read.email`
+- `users:read`
+Once the slack app is created and installed on the slack workspace, you can use the slack bot user oauth token of slack app to setup the slack channel.
```
Comment (dhope-nagesh) [inline advocacy_docs/edb-postgres-ai/console/using/notifications.mdx:76]:
```suggestion

Once the slack app is created and installed on the slack workspace, you can use the slack bot user oauth token of slack app to setup the slack channel.
```

Comment (dhope-nagesh) [reply advocacy_docs/edb-postgres-ai/console/using/notifications.mdx:76]: Screenshot 2024-10-18 at 10 49 44 AM

File: advocacy_docs/edb-postgres-ai/console/using/notifications.mdx [hunk 2]
```diff
@@ -42,12 +47,77 @@ The notifications under the **Projects** preference section are:
...
+
+### Configure Webhook notification channel
+
+To configure Webhook notification channel:
+
+1. Log in to the EDB Postgres AI console.
+1. From the menu under your name in the top-right panel, select **Settings**. 
+1. Select the **Notifications** tab. Options available are Slack, Webhook, and Pagerduty.
+1. Select **Webhook** and provide the details on the pop-up window:
+   1. Host URL of the Webhook
```
Comment (dhope-nagesh) [inline advocacy_docs/edb-postgres-ai/console/using/notifications.mdx:88]:
```suggestion
   1. Host URL of the Webhook
   1. HTTP Method of the Webhook 
```

File: advocacy_docs/edb-postgres-ai/console/using/notifications.mdx [hunk 3]
```diff
@@ -42,12 +47,83 @@ The notifications under the **Projects** preference section are:
...
+1. Log in to the EDB Postgres AI console.
+1. From the menu under your name in the top-right panel, select **Settings**. 
+1. Select the **Notifications** tab. Options available are Webhook, Slack, and Pager Duty.
+1. Select **Webhook** and provide the details on the pop-up window:
+   1. Host URL of the Webhook
+   1. Method of the Webhook - POST or PUT
+   1. Optionally provide any of these authentication parameters 
+      a. For Basic Auth:
+         1. Username and password
+         1. Bearer token
```
Comment (dhope-nagesh) [inline advocacy_docs/edb-postgres-ai/console/using/notifications.mdx:98]:
```suggestion
         1. Username
         1. Password
```

**Review by dhope-nagesh — APPROVED** [review]

**Review by gvasquezvargas — APPROVED** [review]
LGTM

**Pull request merged**